project/configs/experiment/cluster_sweep_example.yaml

```diff
@@ -1,4 +1,5 @@
 # @package _global_
+# yaml-language-server: $schema=../../../.schemas/experiment_cluster_sweep_example_schema.json
```

Comment:
```suggestion
```
This being here means that your vscode settings file isn't valid json (probably a trailing comma somewhere). This is the fallback for the auto-schema thing, in case it can't add the schema to the vscode settings.